action.yml

```diff
@@ -55,7 +58,6 @@ runs:
         path: /usr/local/bin/devbox
         key: ${{ runner.os }}-devbox-${{ env.latest_version }}
 
-    - name: Install devbox cli
```

Comment:
why remove this step?

Reply:
Sorry good catch, that was a mistake. I readded it.